contents/handbook/growth/sales/refunds.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Refunds
+sidebar: Handbook
+showTitle: true
+---
+
+Most of the requests we receive fall under the following scenarios:
+- Learning curve: just got off of the startup plan/new client accidentally used us a lot
+- Unexpected stardom: side project sudden volume spike
+- Under attack: bot spike/abusive user drove traffic which in turn increased posthog usage
+
+## Eligibility Criteria
+Customer must meet the following criteria to get a refund:
+- The request is made within 30 days of the billing date.
+- The customer provides a reasonable explanation for the request, fitting into one of the above scenarios.
+- The account does not show signs of fraudulent activity or abuse of PostHog services.
+
+## Scenarios and Procedures
+
+### Learning Curve
+We issue refunds in this category if this is the first bill >$1 and/or they meet criteria as explained above.
```

Comment:
Add that we should also help them figure out reducing their event count/usage volume if that's the concern